### PR TITLE
Added my profile

### PR DESCRIPTION
**Pull request by rkshaw20**
## Check List (Check all the applicable boxes) 

- [x]  My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2398"></a>

### CODE REVIEW

File: data/rkshaw20.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      "url": "https://www.linkedin.com/in/raj-kishor-shaw-636759144/",
+      "icon": "linkedin"
+    },
+    {
+      
+        "name": "Open Source Blog",
+        "url": "https://hashnode.com/@rajkishorshaw",
+        "icon": "hashnode"
+      
+    }
```
Comment (Pradumnasaraf) [inline data/rkshaw20.json:45]:
```suggestion
    {
      "name": "Open Source Blog",
      "url": "https://hashnode.com/@rajkishorshaw",
      "icon": "hashnode"
    }
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Action is failing due to styling. Please fix that.

**Review by eddiejaoude — APPROVED** [review]
Looks good after reformatting the files 👍

**Comment by rkshaw20** [comment]
Thanks @eddiejaoude and @Pradumnasaraf for helping me, you guys are awesome 😃!

**Pull request merged**